packages/web3-utils/src/random.ts

```diff
@@ -0,0 +1,26 @@
+import { randomBytes as cryptoRandomBytes } from 'crypto';
+
+/**
+ * Returns a random byte array by the given bytes size
+ *
+ * @param {Number} size
+ * @returns {Buffer}
+ */
+ export const randomBytes = (byteSize: number): Buffer => {
+	const randomValues =
+		typeof window !== 'undefined' && window.crypto && window.crypto.getRandomValues
```

Comment:
```suggestion
		window !== undefined && window.crypto && window.crypto.getRandomValues
```

Nit picky, don't necessary have to commit this, just used to checking `undefined` like this

Reply:
[This](https://stackoverflow.com/a/4725697) is actually a really good point as why we should use `typeof window !== 'undefined'` here